src/helpers/figures/charts/bar_chart.ts

```diff
@@ -303,8 +303,8 @@ export function createBarChartRuntime(chart: BarChart, getters: Getters): BarCha
     const dataset: ChartDataset<"bar", number[]> = {
       label,
       data,
-      borderColor: BORDER_CHART_COLOR,
-      borderWidth: 1,
```

Comment:
A small test would be nice, so we do'nt break this again :slightly_smiling_face: